### ISSUE TITLE
[Creature/DB]Speaker Mar'grom dont say anything in combat

### ISSUE THREAD

**Issue by Remberance** [azerothcore/azerothcore-wotlk#3507]
##### CURRENT BEHAVIOUR:
when you find Speaker Mar'grom in Outland, combat with him, he will not say anything.

##### EXPECTED BLIZZLIKE BEHAVIOUR:
he says: My message of unity is meant for the Ogre clans, not for you, $R!  while in combat. (picture from NGA, thanks to author)
![image](https://user-images.githubusercontent.com/4078506/94354636-bf147d80-00af-11eb-884c-244b2e4f3706.png)

##### STEPS TO REPRODUCE THE PROBLEM:

1. .go c 151923/151924/151925/151926/151927  until find him.
2. combat with him.
3. apply SQL in EXTRA NOTES.
4. restart, relog, combat with him.

##### EXTRA NOTES:

```sql
UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 18693;

DELETE FROM `smart_scripts` WHERE (`entryorguid` = 18693) AND (`source_type` = 0) AND (`id` IN (4));
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(18693, 0, 4, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Speaker Mar\'grom - In Combat - say 0');

DELETE FROM `creature_text` WHERE (`CreatureID` = 18693) AND (`GroupID` = 0);
INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
(18693, 0, 0, 'My message of unity is meant for the Ogre clans, not for you, $R!', 12, 0, 100, 19884, 0, 'Speaker Mar\'grom - In Combat - say 0');
```

##### AC HASH/COMMIT:
1910b4ab35bf6ecc65f02df3e06cba4b6805318d

##### OPERATING SYSTEM:
Windows 10

##### MODULES:
ahbot bgreward buffer beastmaster trangmog

**Comment by FrancescoBorzi** [comment]
@Remberance I've edited your post to better format the SQL code, FYI:

![image](https://user-images.githubusercontent.com/75517/94367203-6cea5100-00dd-11eb-920c-86e53abb330a.png)

**Comment by Remberance** [comment]
> @Remberance I've edited your post to better format the SQL code, FYI:
> 
> ![image](https://user-images.githubusercontent.com/75517/94367203-6cea5100-00dd-11eb-920c-86e53abb330a.png)

okok, thank you. i will post some Sai lik above SQL here soon.

**Comment by Si1ker** [comment]
Fix included in the issue is not the right one, is incomplete and deleting already existing SAI which is needed.

**Comment by Remberance** [comment]
#3509  close issus

**Comment by FrancescoBorzi** [comment]
@Remberance the issue will automatically be closed once the PR is merged into master. We don't close issues until the solution is on master.
